exercises/binary-search-tree/canonical-data.json

```diff
@@ -1,186 +1,189 @@
 {
-    "exercise": "binary-search-tree",
-    "comments": [
-        "Each test case assumes an empty/new tree.",
-        "As per exercism/problem-specifications#996 key 'treeData' counts as an input",
-        "to test generators.",
-        "The key 'treeData' represents an array of numbers for which the data should be ",
-        "inserted/added to the tree as it appears in the array from left to right.",
-        "e.g. treeData: ['2', '1', '3', '6', '7', '5']",
-        "Insert 2. Insert 1. Insert 3. Insert 6, so on...",
-        "This canonical-data does not restrict the data type of array elements to either",
-        "strings or integers.",
-        "The key 'expected' represents tree state as JSON object of schema :",
-        "{",
-        "    'title':'nodeObject',",
-        "    'type':'object',",
-        "    'properties':{",
-        "        'data':{",
-        "            'type':'string'",
-        "        },",
-        "        'left':{",
-        "            'type':'nodeObject'",
-        "        },",
-        "        'right':{",
-        "            'type':'nodeObject'",
-        "        }",
-        "    },",
-        "    'required':['data', 'left', 'right']",
-        "}"
-    ],
-    "cases": [{
-            "uuid": "e9c93a78-c536-4750-a336-94583d23fafa",
-            "description": "data is retained",
-            "property": "data",
-            "input": {
-                "treeData": ["4"]
+  "exercise": "binary-search-tree",
```

Comment:
Could someone who knows this exercise verify that this is indeed just a formatting change, please?